src/map.cpp

```diff
@@ -620,6 +617,10 @@ void map::move_vehicle( vehicle &veh, const tripoint &dp, const tileray &facing
             veh.skidding = true;
             veh.turn( coll_turn );
         }
+        // accept new position
+        // if submap changed, we need to process grid from the beginning.
+        // TODO: Allow vehicles to get displaced vertically
```

Comment:
I don't get the comments here. What do they have to do with the new code? Displacing vertically is now possible and that line should probably go. Similar the other two (why even bring up the submaps?)